config/config.txt

```diff
@@ -213,10 +213,10 @@ CHECK_RANDOMIZER
 # SERVER ss13.example.com:2506
 
 ## forum address
-# FORUMURL http://tgstation13.org/phpBB/index.php
+# FORUMURL https://discord.gg/BbsYDrR
 
 ## Wiki address
-# WIKIURL http://www.tgstation13.org/wiki
+# WIKIURL https://www.patreon.com/baddeathclaw
```

Comment:
good wiki link

Reply:
wiki was changed to patreon in the top right, we dont have a wiki and all attempts to make one falter, the slot is better put to use to something that can benifiet the server

also im lazy and didnt want to change the word "wiki" to patreon 300 times

Reply:
https://github.com/BadDeathclaw/TG-Claw/wiki yeah I know it's shit and empty.

You can also use mass find/mass replace using atom or your favorite editing program to just mass-replace every instance of a word wherever you want, honestly. It's really easy.

Reply:
I'll go through the trouble of making a seperate button for the wiki if and when we add it or actually have one

Until then it's just wasted time when we could use the wiki url thats already in for something productive.